src/main/java/com/google/sps/servlets/SchedulingServlet.java

```diff
@@ -31,7 +45,33 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        response.setContentType("plain/text");
-        response.getWriter().println("To be implemented");
+        String tutorID = request.getParameter("tutorID");
+        String start = request.getParameter("start");
+        String end = request.getParameter("end");
+        String studentEmail = request.getParameter("studentEmail");
+        String subtopics = request.getParameter("subtopics");
+        String questions = request.getParameter("questions");
```

Comment:
Maybe use Optionals to make default values for these Strings?

Reply:
These all already have null as their default values, which I think should be the default if a user has not filled out those fields.